tests/bluetooth/host/keys/mocks/keys_help_utils.c

```diff
@@ -17,6 +17,24 @@ void clear_key_pool(void)
 	memset(key_pool, 0x00, sizeof(struct bt_keys) * CONFIG_BT_MAX_PAIRED);
 }
 
+void fill_key_pool_by_id_addr(const struct id_addr_pair src[], int size, struct bt_keys *refs[])
+{
+	uint8_t id;
+	bt_addr_le_t *addr;
+	struct id_addr_pair const *params_vector;
+
+	zassert_true(check_key_pool_is_empty(),
+		     "List isn't empty, make sure to run this test just after a fresh start");
```

Comment:
This function should only be in the test suite setup.

Reply:
Ok, I wasn't fully satisfied with it as well.

My point was that this function will usually be used while setting up the test and it's behavior won't be checked.
So, I wasn't sure if this function should implicitly clear the keys pool to ensure correct behavior or it shouldn't alter the current state of the keys pool.